### PR TITLE
Add #pop functionality

### PR DESCRIPTION
**Pull request by joshuaborn**
This is pretty straightforward.

### CODE REVIEW

File: src/router.js [hunk 1]
```diff
@@ -97,4 +97,13 @@ export default class Router {
   forward() {
     window.history.forward();
   }
+
+  pop() {
+    if (this.currentPath() !== '/') {
```
Comment (jdlehman) [inline src/router.js]: I am not sure if this implementation assumes too much.

what about something like:

```js
var matches = this.currentPath().match(/.+(\/.+\/?)/);
if (matches) {
  this.navigate(matches[1]);
}
```

Comment (joshuaborn) [inline src/router.js]: I was going to handle the lack of trailing slash case, but I saw another test that ensured currentPath returns a trailing slash, so I assumed it.

Do you think `/test/` shouldn't go to `/`?

File: docs/router.md [hunk 2]
```diff
@@ -92,3 +92,14 @@ Calls `window.history.forward`
 ```js
 router.forward();
 ```
+
+## forward
```
Comment (jdlehman) [inline docs/router.md]: Copy paste error :stuck_out_tongue:

Comment (joshuaborn) [inline docs/router.md]: Good catch!

File: src/router.js (hunk 1 above)
Comment (jdlehman) [inline src/router.js]: yeah we can do 

```js
this.navigate(`${matches[1]}/`);
```

**Comment by joshuaborn** [comment]
@jdlehman Changes are incorporated and tests are passing.

**Pull request merged**